app/src/main/res/layout/item_list_row.xml

```diff
@@ -95,6 +95,286 @@
                         app:textEndPadding="0dp"
                         app:textStartPadding="0dp" />
 
+                    <com.google.android.material.chip.Chip
```

Comment:
also, we should use the tag.xml from the fragment_xml branch with the styles later. not on you, i havent pushed it to dev yet

Reply:
If we use tag.xml we won't need hardcoded .xml code for each tag, correct? We could then generate as many chips as needed?